mappings/net/minecraft/entity/ProjectileUtil.mapping

```diff
@@ -1 +1,5 @@
 CLASS avm net/minecraft/entity/ProjectileUtil
+	METHOD a getHandPossiblyContaining (Laih;Lbau;)Lags;
```

Comment:
This name is a bit long. Maybe `getHandPossiblyHolding`, or even `guessHandHolding` ?